.circleci/circle_trigger.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
```

Comment:
Note this is sourced from https://github.com/labs42io/circleci-monorepo so doesn't need an in depth review